diff --git a/data/io.github.seadve.Mousai.appdata.xml.in b/data/io.github.seadve.Mousai.appdata.xml.in
index 6ca5bd6..868672a 100644
--- a/data/io.github.seadve.Mousai.appdata.xml.in
+++ b/data/io.github.seadve.Mousai.appdata.xml.in
@@ -32,13 +32,13 @@
 
   <screenshots>
     <screenshot type="default">
-      <image>https://raw.githubusercontent.com/SeaDve/Mousai/main/screenshots/Mousai-screenshot.png</image>
+      <image>https://raw.githubusercontent.com/SeaDve/Mousai/cc32571c52c09983cd101c599a5dee5516a11d02/screenshots/Mousai-screenshot.png</image>
     </screenshot>
     <screenshot>
-      <image>https://raw.githubusercontent.com/SeaDve/Mousai/main/screenshots/Mousai-screenshot2.png</image>
+      <image>https://raw.githubusercontent.com/SeaDve/Mousai/cc32571c52c09983cd101c599a5dee5516a11d02/screenshots/Mousai-screenshot2.png</image>
     </screenshot>
     <screenshot>
-      <image>https://raw.githubusercontent.com/SeaDve/Mousai/main/screenshots/Mousai-screenshot3.png</image>
+      <image>https://raw.githubusercontent.com/SeaDve/Mousai/cc32571c52c09983cd101c599a5dee5516a11d02/screenshots/Mousai-screenshot3.png</image>
     </screenshot>
   </screenshots>
 
